easybuild/easyconfigs/l/LIGGGHTS/LIGGGHTS-3.8.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
```

Comment:
Please add as comment the name of the person in charge of this manual installation and the ticket number